website/docs/Pipelines/reblock_gvcf_spec.md

```diff
@@ -0,0 +1,68 @@
+---
+sidebar_position: 14
+---
+
+# Reblocked GVCF Technical Overview
+The following is a technical overview of the current reblocked GVCF schema (gnomADv4) as well descriptions of the previous GVCF schemas.
```

Comment:
```suggestion
The following is a technical overview of the current reblocked GVCF schema for the Whole Genome and Exome Germline Pipelines (v3.0.0 and newer) as well descriptions of the previous GVCF schemas. Published 11/18/2021.
```

Doesn't have to be this exact wording but I think it's good to note when it's current as of and when it takes effect in the pipelines.